app/views/console/users/index.phtml

```diff
@@ -21,10 +21,11 @@ $smtpEnabled = $this->getParam('smtpEnabled', false);
 
             <h2>Users</h2>
 
+            <!-- TODO: New queries syntax doesnt work. Needs to be fixed -->
             <form class="box padding-small margin-bottom search"
                 data-service="users.list"
                 data-event="submit"
-                data-param-search="{{router.params.search}}"
+                data-param-queries="search.search('{{router.params.search}}')"
```

Comment:
This is still open, I could not yet figure out how to do it in our JS framework. Torsten will investigate it later.

Reply:
We decided to address this in separate UI PR, once we have API changes finished. It will make PR review easier.

I have removed changes to PHTML files.